src/argilla/client/feedback/schemas.py

```diff
@@ -168,10 +367,80 @@ def update_settings(cls, values: Dict[str, Any]) -> Dict[str, Any]:
 
 
 class LabelQuestion(_LabelQuestion):
```

Comment:
No need to add docstrings to this `pydantic.BaseModel` since we already have a docstrings for the `pydantic.BaseModel`s inheriting from this one, as this one shouldn't be used as protected (we can also make it private)